app_flutter/lib/build_dashboard_page.dart

```diff
@@ -119,6 +119,29 @@ class BuildDashboard extends StatelessWidget {
           title: statusTable[buildState.isTreeBuilding],
           backgroundColor: colorTable[buildState.isTreeBuilding],
           actions: <Widget>[
+            DropdownButton<String>(
+                value: buildState.currentBranch,
+                icon: Icon(
+                  Icons.arrow_downward,
+                  color: Colors.white,
```

Comment:
in theory the color should be picked up automatically by the ambient theme, something like how i made it happen for the SIGN IN button. cc @HansMuller for advice.

Reply:
Thanks! I am going to submit on green to enable testing of the branching release strategy. Will follow up with smoothing this out next week.